boa_engine/src/context/mod.rs

```diff
@@ -656,6 +661,12 @@ impl Context<'_> {
         std::mem::swap(&mut self.realm, realm);
     }
 
+    /// Get and increment the parser identifier.
```

Comment:
I would change the comment to "increment and get the parser identifier", since we first increment it, and then return it. To make sure that the users of the function are not surprised ;)

Reply:
This should be fixed now. @Razican if you want to give it another review :)